nautobot/extras/jobs.py

```diff
@@ -189,6 +191,24 @@ def before_start(self, task_id, args, kwargs):
             )
             raise RunJobTaskFailed(f"Job {self.job_model} is not enabled to be run!")
 
+        if self.job_model.is_singleton and not self.celery_kwargs.get("nautobot_job_ignore_singleton_lock", False):
```

Comment:
In the case where the job *is* a singleton *and* the `is_running` key is set, *but* `ignore_singleton_lock` is set, I wonder if it would be good to log a log entry to the effect of "Ignoring singleton lock because ignore_singleton_lock is set"?

Reply:
Good idea, done